packages/wdio-cli/src/utils.ts

```diff
@@ -611,21 +601,36 @@ export function runProgram (command: string, args: string[], options: SpawnOptio
  * create package.json if not already existing
  */
 export async function createPackageJSON (parsedAnswers: ParsedAnswers) {
-    if (!parsedAnswers.createPackageJSON) {
-        return
+    const packageJsonExists = await fs.access(path.resolve(process.cwd(), 'package.json')).then(() => true, () => false)
+
+    // Use the exisitng package.json if it already exists.
+    if (packageJsonExists) { return }
```

Comment:
```suggestion
    if (packageJsonExists) {
        return
    }
```

Reply:
This wasn't formatted for me automatically and ESLint doesn't have a problem with it.
Are the EditorConfig rules supposed to do that?